src/appShell/App/Container.tsx

```diff
@@ -68,9 +69,27 @@ export default class Container extends React.Component<IContainerProps, {}> {
     }
 
     get appContext(): IAppContext {
+        const downloadConfig: string = getServerConfig()
```

Comment:
@kalletlak, will this be backward compatible?  if they had it as false in portal.properties, it seems it would now default to true?

Reply:
Yeah, looks like not backward compatible. I'll discuss and let you know.